src/routes/views/app/LoginView.tsx

```diff
@@ -0,0 +1,27 @@
+import InputField from '../../../components/InputField'
+import InputFieldPassword from '../../../components/InputFieldPassword'
+import Button from '../../../components/Button'
+import { Link } from 'react-router-dom'
+import { routes } from '../../routes'
+
+const LoginView = () => {
+    return (
+        <div>
```

Comment:
I would extract this into a component called `LoginForm`. We also need an entity like `Credentials` which contains the username and password. The `Credentials` could then be passed to the `LoginForm` as a prop.